TACC/Core-CMS-Custom
# feat(example): improve example app, add upgrade doc

## Overview

Update `example_cms` CMS tag to be v3.12 (latest).

## Related

None.

## Changes

- add "Upgrade Project" doc
- update CMS image tag
- rename custom app to `example_app`
	<sup>This ensures changes are tested without accidentally relying on Core-CMS's `custom_example`.</sup>
- move nested images up to `/img` dir
- add and load stylesheet
	<sup>To show assets can be either project-specific or app-specific.</sup>
- set `TACC_CORE_STYLES_VERSION = 2`
- add URL to load example app

## Testing

1. Verify `site.css` has `v3.12` in top-of-file comment.
2. Verify logo is default "Portal" logo.
3. Verify favicon is default favicon (white star on blue field).
4. Verify http://localhost:8000/example_app/ loads example app.
5. Verify example app title is "Example App" in **green** text.

## UI

![example_cms example_app page screenshot](https://github.com/TACC/Core-CMS-Custom/assets/62723358/b0a417ff-e48d-4235-935e-a10ff9ed09a2)
